swim/ping_sender.go

```diff
@@ -132,6 +132,35 @@ func (p *pingSender) MakeCall(ctx json.Context, res *ping) <-chan error {
 	return errC
 }
 
+// sendPingWithChanges sends a special ping to the target with the given changes.
+// In normal pings the disseminator is consulted to create issue the changes,
+// this is not the case in this function. Only the given changes are transmitted.
```

Comment:
Maybe we should generalize the "normal" ping and call that with arguments to send over? In the standard case, that would be the changes from disseminator. In partition case, it would be suspects.

I`m afraid that if we introduce another "special type of ping", we will keep baking "special versions of ping", which will not look great.